packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -486,13 +496,21 @@ export default function() {
             if (specifiers.length) {
               let uid;
 
-              if (hasSingleDefaultImport(specifiers) && this.modulesType === "commonjs") {
+              if (
+                hasSingleDefaultImport(specifiers) &&
+                this.modulesType === "commonjs"
+              ) {
                 uid = addRequire(source, maxBlockHoist, {
-                  inline: this.addHelper("interopRequireDefault")
+                  inline: this.addHelper("interopRequireDefault"),
                 });
-              } else if (specifiers.length === 1 && t.isImportNamespaceSpecifier(specifiers[0]) && this.modulesType === "commonjs") {
+              } else if (
+                specifiers.length === 1 &&
+                t.isImportNamespaceSpecifier(specifiers[0]) &&
+                this.modulesType === "commonjs"
+              ) {
                 uid = addRequire(source, maxBlockHoist, {
-                  inline: this.addHelper("interopRequireWildcard")
+                  inline: this.addHelper("interopRequireWildcard"),
+                  identifier: specifiers[0].local,
```

Comment:
added this to reuse for `import * as a from './a'` to use `a` as the identifier instead of 

```
const ref = opts.identifier
              ? opts.identifier
              : path.scope.generateUidIdentifier(
                basename(source, extname(source))
              );
```